.github/workflows/_build_docker.yaml

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2022 - 2023, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+# This reuseable workflow builds and tests the Macaron Docker image.
+
+name: Build and push Docker image
+on:
+  workflow_call:
+    inputs:
+      artifact-name:
+        required: true
+        type: string
+      artifact-sha256:
+        required: true
+        type: string
+permissions:
+  contents: read
+
+jobs:
+  build-docker:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: read
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+      with:
+        fetch-depth: 0
+
+    - name: Download artifact
+      uses: actions/download-artifact@9782bd6a9848b53b110e712e20e42d89988822b7 # v3.0.1
+      with:
+        name: ${{ inputs.artifact-name }}
+        path: dist
+
+    # Verify hashes by first computing hashes for the artifacts and then comparing them
+    # against the hashes for the artifact.
+    - name: Verify the artifact hash
+      env:
+        ARTIFACT_HASH: ${{ inputs.artifact-sha256 }}
+      run: |
+        set -euo pipefail
+        cd dist
+        echo "Hash of package should be $ARTIFACT_HASH."
+        echo "$ARTIFACT_HASH" | base64 --decode | sha256sum --strict --check --status || exit 1
+        cd ${{ github.workspace }}
+
+    # We need to login to access the private macaron-base image.
+    # TODO: remove it when macaron-base is public.
+    - name: Login to ghcr.io
+      run: docker login ghcr.io --username ${{ github.actor }} --password ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Build the Docker image
+      env:
+        IMAGE_NAME: ghcr.io/oracle-samples/macaron
+      run: ./scripts/dev_scripts/build_docker.sh "${IMAGE_NAME}" ${{ github.workspace }} ${{ inputs.release-tag }}
+
+    - name: Test the Docker image
+      env:
+        # This will be picked up by run_macaron.sh.
```

Comment:
```suggestion
        # This environment variable will be picked up by run_macaron.sh.
```

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/76/commits/d2fe70dc8bc12b52071d18e885bf63b86f9dcc25